orange/src/main/java/com/hartwig/hmftools/orange/report/interpretation/Variants.java

```diff
@@ -82,6 +82,23 @@ public static String hotspotField(@NotNull VariantEntry variant)
         }
     }
 
+    @NotNull
+    public static String biallelicLikelihoodField(@NotNull VariantEntry variant)
+    {
+        if(variant.biallelicProbability() != null)
+        {
+            return PERCENTAGE_FORMAT.format(variant.biallelicProbability() * 100);
+        }
+        else if(variant.biallelic())
+        {
+            return "Yes";
+        }
+        else
+        {
+            return "No";
+        }
```

Comment:
They wouldn't be. I at first thought it would make sense the null case since it is a possibility in the Orange datamodel.
On second thought, this is a bit too confusing, since it would never actually happen in Orange, and this makes it look like it can.